scripts/purgecss.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+
+const Purgecss = require('purgecss');
+const fetch = require('node-fetch');
+const { writeFileSync } = require('fs');
+const { join } = require('path');
+
+const main = async () => {
```

Comment:
Do you think you could add some comments ahead of this just explaining what informed the strategy we're using here?

Reply:
✔️